llama/runner/image.go

```diff
@@ -76,17 +76,40 @@ func (c *ImageContext) NewEmbed(llamaContext *llama.Context, data []byte, aspect
 	embed, err := c.findImage(hash)
 	if err != nil {
 		if c.mllama != nil {
-			embed = c.mllama.NewEmbed(llamaContext, data, aspectRatioId)
+			embed, err = c.mllama.NewEmbed(llamaContext, data, aspectRatioId)
```

Comment:
```suggestion
			embed, err := c.mllama.NewEmbed(llamaContext, data, aspectRatioId)
```

New scope, new vars.

Reply:
Or is this shadowing? If so, wrap in a function and return the `embed, err` pair so we can clearly see that this is what is happening?

Reply:
`embed` is the same variable - check cache and, if not found, generate.

It probably would have been clearer to return nil from `findImage` rather than an error but I don't want to mix in unrelated changes into a patch, especially at this stage of the release process.